### DISCUSSION TITLE
Ability to group elements, have a parent with several child elements?

### DISCUSSION THREAD

**Discussion by butterlym** [clientIO/joint#2835] [Q&A] [answered]
### Introduction

Thank you for a great product, 
Is it possible to group elements under a parent, e.g. an Element that forms a boundary and all elements within(enclosed by) that boundary are children elements, and so on and so forth. 

### Steps to reproduce

Create large element, pop other smaller elements inside, all small elements are now children of the larger element

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment, marked as answer]
The feature is called `embedding`. Here are some [examples](https://www.jointjs.com/demos?feature=Embedding).
